.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Automatically request reviews when a pull request changes any owned files
+# More information: https://github.com/blog/2392-introducing-code-owners 
+
+*.groovy @dotnet/roslyn-infrastructure
+.github/ @dotnet/roslyn-infrastructure
+build/ @dotnet/roslyn-infrastructure
+samples/ @jmarolf
```

Comment:
Should we have a team for this, especially given it's flu season? :smile:

Reply:
We can. You want to be on it 😃

Reply:
@jasonmalinowski I don't have permissions to create a group.  Who do I talk to about that?

Reply:
Does https://github.com/orgs/dotnet/new-team not work? I thought everybody had permissions.

Reply:
that worked for me! I had different, old, instructions

Reply:
@jmarolf I guess I was imagining an SDK team that owns both samples and templates.

Reply:
Sure, that can work. Templates has been only me for a while.  Don't know who would review that.

Reply:
The team can be you for now. :-)